src/ontology/imports/uo-extracted.owl

```diff
@@ -124,6 +138,7 @@
     <!-- http://purl.obolibrary.org/obo/UO_0000000 -->
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/UO_0000000">
+        <obo:IAO_0000115>&quot;A unit of measurement is a standardized quantity of a physical quality.&quot; [Wikipedia:Wikipedia]</obo:IAO_0000115>
```

Comment:
By converting UO's `rdfs:comment` to `definition` now `unit` has two definitions:
![grafik](https://github.com/OpenEnergyPlatform/ontology/assets/36884905/a99e0e86-242e-4267-aeb1-da2c12ee2dae)

Reply:
A quick fix might be to manually convert in this single case back to `rdfs:comment`. Then we are sure the oeo-viewer and other tools use our definition.

The uo-import script can then later (e.g. after the release) be adapted to make an exception for UO:0000000.